apps/slack-node/src/services/auth-service.js

```diff
@@ -1,3 +1,4 @@
+// Old auth
```

Comment:
Make comment more descriptive - 

```suggestion
// Helper functions if you want to implement OAuth in Slack from scratch, instead of using the Credentials App Feature which handles it for you. 
```